src/index.html

```diff
@@ -4,10 +4,408 @@
+              </div>
+            </div>
+            <div class="place__description place__description--margin">
+              Audio sunglasses
+            </div>
+          </div>
+          <div class="button">
+            <div class="grid grid--mobile">
+              <div class="grid__item grid__item--desktop-5-8 grid__item--tablet-2-5">
+                <a href="#gallery" class="button__link">
+                  View all
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="testimonials page__section" id="testimonials">
+        <div class="container">
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--tablet-1-3 grid__item--desktop-1-4">
+              <h2 class="page__section-title testimonials__title">
+                Why buy direct
+                <br>
+                from Bose
+              </h2>
+            </div>
+            <div class="grid__item  grid__item--tablet-4-6 grid__item--desktop-6-12">
+              <h3 class="page__section-subtitle testimonials__subtitle">
+                Free 2-day shipping and returns
+              </h3>
+
+              <h3 class="page__section-subtitle testimonials__subtitle">
+                90-day risk-free trial
+              </h3>
+
+              <h3 class="page__section-subtitle testimonials__subtitle">
+                World class customer service
+              </h3>
+
+              <h3 class="page__section-subtitle testimonials__subtitle">
+                My Bose account management
+              </h3>
+
+              <p class="page__section-text testimonials__text grid__item grid__item--tablet-4-6 grid__item--desktop-6-10">
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="contacts page__section" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--desktop-1-5 grid__item--tablet-1-3">
+              <form
+                method="post"
+                action="#"
+                class="contacts__form"
+                onsubmit="return false"
+              >
+                <input
+                  type="text"
+                  class="contacts__form-input"
+                  placeholder="Name"
+                  required
+                >
+                <input
+                  type="email"
+                  class="contacts__form-input"
+                  placeholder="Email"
+                  required
+                >
+
+                <textarea
```

Comment:
make sure to use the same font as for the rest of the page
<img width="207" alt="Monosnap Miami 2022-11-23 09-42-10" src="https://user-images.githubusercontent.com/39308717/203494103-2b6e7e6c-c0c6-4b0e-9ac6-268a7ee7118a.png">